core/org.openjdk.jmc.flightrecorder/src/main/java/org/openjdk/jmc/flightrecorder/jdk/JdkAttributes.java

```diff
@@ -792,9 +793,27 @@ public <U> IMemberAccessor<IMCThread, U> customAccessor(IType<U> type) {
 	public static final IAttribute<String> SHUTDOWN_REASON = attr("reason", //$NON-NLS-1$
 			Messages.getString(Messages.ATTR_SHUTDOWN_REASON), Messages.getString(Messages.ATTR_SHUTDOWN_REASON_DESC),
 			PLAIN_TEXT);
-	public static final IAttribute<IQuantity> CLASSLOADER_LOADED_COUNT = attr("loadedClassCount", //$NON-NLS-1$
+	public static final IAttribute<Number> CLASSLOADER_LOADED_COUNT_NUMBER = attr("loadedClassCount", //$NON-NLS-1$
 			Messages.getString(Messages.ATTR_CLASSLOADER_LOADED_COUNT),
-			Messages.getString(Messages.ATTR_CLASSLOADER_LOADED_COUNT_DESC), NUMBER);
+			Messages.getString(Messages.ATTR_CLASSLOADER_LOADED_COUNT_DESC), RAW_NUMBER);
+
+	public static final IAttribute<IQuantity> CLASSLOADER_LOADED_COUNT = Attribute.canonicalize(
+			new Attribute<IQuantity>("loadedClassCount", Messages.getString(Messages.ATTR_CLASSLOADER_LOADED_COUNT), //$NON-NLS-1$
+					Messages.getString(Messages.ATTR_CLASSLOADER_LOADED_COUNT_DESC), NUMBER) {
+				@Override
+				public <U> IMemberAccessor<IQuantity, U> customAccessor(IType<U> type) {
+
+					final IMemberAccessor<Number, U> accessor = CLASSLOADER_LOADED_COUNT_NUMBER.getAccessor(type);
+					return accessor == null ? null : new IMemberAccessor<IQuantity, U>() {
+						@Override
+						public IQuantity getMember(U i) {
+							Number countNumber = accessor.getMember(i);
+							return countNumber == null ? null : UnitLookup.NUMBER_UNITY.quantity(countNumber);
+						}
+					};
+				}
+			});
+
 	public static final IAttribute<IQuantity> CLASSLOADER_UNLOADED_COUNT = attr("unloadedClassCount", //$NON-NLS-1$
```

Comment:
**question:** Could this be useful to have that for similar attributes like `unloadedClassCount`.

Reply:
yes it resolves issue for all such attributes.

Here is the example of a JFR where unloadedClassCount data is present.

Before the change:
<img width="502" alt="image" src="https://github.com/user-attachments/assets/d465eb61-12b9-4946-99ec-325154a7104e">

After the change:
<img width="499" alt="image" src="https://github.com/user-attachments/assets/0e76022f-708f-4ce9-b449-3a34cd7aa4da">